src/Payments/GPay.res

```diff
@@ -87,8 +89,14 @@ let make = (
       if dict->Js.Dict.get("gpayResponse")->Belt.Option.isSome {
         let metadata = dict->getJsonObjectFromDict("gpayResponse")
         let obj = metadata->getDictFromJson->itemToObjMapper
+        let gPayBody = {
+          let body = PaymentBody.gpayBody(~payObj=obj, ~connectors)
+          PaymentUtils.isAppendingCustomerAcceptance(isGuestCustomer, list.payment_type)
```

Comment:
Instead of checking isAppendingCustomerAcceptance here itself. Please create a function which accepts body as well and returns the final body with or without appending customerAcceptance

Reply:
Updated